packages/cli/test/integration/credentials.test.ts

```diff
@@ -48,25 +53,24 @@ afterAll(async () => {
 test('POST /credentials should create cred', async () => {
 	const ownerShell = await testDb.createUserShell(globalOwnerRole);
 
-	const authOwnerAgent = utils.createAgent(app, { auth: true, user: ownerShell });
 	const payload = randomCredentialPayload();
 
-	const response = await authOwnerAgent.post('/credentials').send(payload);
+	const response = await authAgent(ownerShell).post('/credentials').send(payload);
 
 	expect(response.statusCode).toBe(200);
 
 	const { id, name, type, nodesAccess, data: encryptedData } = response.body.data;
 
 	expect(name).toBe(payload.name);
 	expect(type).toBe(payload.type);
-	expect(nodesAccess[0].nodeType).toBe(payload.nodesAccess[0].nodeType);
+	expect(nodesAccess[0].nodeType).toBe(payload.nodesAccess![0].nodeType);
```

Comment:
Nit: We should use Jest's `fail()` instead of non-null asserting.

Reply:
it'll be in Val's PR #3672